.github/workflows/test-hygiene.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       - name: Print Git Status
         run: echo $(git status) && echo $(git diff)
       - name: Assert Notebooks Match
-        run: git status | grep "nothing to commit, working tree clean"
+        run:  git status --untracked-files=all demo/ | grep "nothing to commit, working tree clean"
```

Comment:
Changed this to just check the `demo` dir. before was failing because `venv/` was in untracked files.